File: Java/Logic-1/alarmClock.java
public String alarmClock(int day, boolean vacation) {
  if (day >= 1 && day <= 5 && !vacation) {
    return "7:00";
  } else if (day == 6 && !vacation || day == 0 && !vacation || day >= 1 && day <= 5) {
    return "10:00";
  }
  return "off";
}

File: Python/Logic-1/alarm_clock.py
def alarm_clock(day, vacation):
  
  if vacation:
    if day in range(1, 6):
      return "10:00"
    else:
      return "off"
  elif not vacation:
    if day in range(1, 6):
      return "7:00"
    else:
      return "10:00"
